deepcell_applications/utils_test.py

```diff
@@ -26,10 +26,14 @@
 """Tests for deepcell_applications.utils"""
 
 import copy
+import os
 
 import numpy as np
+import skimage.io as io
 
+import argparse
 import pytest
+import tempfile
```

Comment:
`argparse` and `tempfile` are built-in libraries and should get grouped with `copy` and `os`.